configs/diffusion_hippocampus.py

```diff
@@ -23,13 +23,17 @@ def get_context(device, variables, fold=0, predict_hbt=False, **kwargs):
         ImageLoader(glob_pattern="mean_dwi.*", image_name='mean_dwi', image_constructor=tio.ScalarImage),
         ImageLoader(glob_pattern="md.*", image_name='md', image_constructor=tio.ScalarImage),
         ImageLoader(glob_pattern="fa.*", image_name='fa', image_constructor=tio.ScalarImage),
+        ImageLoader(glob_pattern="full_dwi.*", image_name='full_dwi', image_constructor=tio.ScalarImage),
```

Comment:
Does this need `channels_last=True` or will the channels be in the correct order?